tests/beta/test_default_tensor_tf.py

```diff
@@ -523,14 +523,14 @@ def circuit(x):
         dev2 = qml.device("default.qubit", wires=3)
 
         circuit1 = QNode(circuit, dev1, diff_method=diff_method, h=1e-7)
-        circuit2 = QNode(circuit, dev2, diff_method=diff_method, h=1e-7)
+        circuit2 = QNode(circuit, dev2, diff_method="parameter-shift", h=1e-7)
```

Comment:
@antalszava How come this was changed to parameter shift? Previously this test compared the result of default.tensor.tf and default.qubit for the same diff method

Reply:
These changes came due to #552 (adding the `"classical"` and `"device"` diff methods):
The `"device"` diff method was added which is only supported by `default.tensor.tf`. This is not supported by `default.qubit`. Due to parameterization the `diff_method="device"` case fails with `The default.qubit device does not provide a native method for computing the jacobian.`).

We could compare the same `diff_method` for every case except for `diff_method="device"`. @josh146 would you have any further notes on this one?

Reply:
Nope, that sums it up @antalszava! Essentially, the QNode decorator can now create `PassthruQNodes` and `DeviceJacobianQNodes` as of #552, which are not supported by `default.qubit`.

Since this integration is simply to check that the derivatives of `default.tensor.tf` agree with `default.qubit`, I don't think it requires having the diff method of default qubit changing.